eZ/Publish/API/Repository/Tests/_fixtures/Legacy/data/test_data.yaml

```diff
@@ -824,8 +824,8 @@ ezurlalias_ml_incr:
   - { id: 41 }
 ezurlwildcard: {  }
 ezuser:
-  - { contentobject_id: 10, email: nospam@ez.no, login: anonymous, password_hash: $2y$10$35gOSQs6JK4u4whyERaeUuVeQBi2TUBIZIfP7HEj7sfz.MxvTuOeC, password_hash_type: 7 }
-  - { contentobject_id: 14, email: spam@ez.no, login: admin, password_hash: $2y$10$FDn9NPwzhq85cLLxfD5Wu.L3SL3Z/LNCvhkltJUV0wcJj7ciJg2oy, password_hash_type: 7 }
```

Comment:
Note that this change affects integration tests in migration bundle. Please ping me if/when this change goes through.